test/native/helpers.js

```diff
@@ -122,7 +123,7 @@ export function waitFor(
 		).then( () => {
 			if ( ! result ) {
 				reject(
-					`waitFor timed out after ${ timeout }ms for callback:\n${ cb }`
+					`waitFor timed out after ${ timeout }ms for callback:\n${ cb }\n${ lastError.toString() }`
```

Comment:
While testing I realized that errors not directly related to timeout, like syntax errors, were omitted. For this reason, I updated this function by displaying now the last error that caused the timeout:

**Example 1 - Querying for an unexisting text:**
<img width="713" alt="Screenshot 2022-01-24 at 11 17 30" src="https://user-images.githubusercontent.com/14905380/150770619-b6dfb388-ad85-46a0-9c61-0e0e36268dfd.png">

**Example 2 - Reference error:**
<img width="729" alt="Screenshot 2022-01-24 at 12 00 45" src="https://user-images.githubusercontent.com/14905380/150771127-da6fdc31-f7d3-495b-a9fb-6cbf9b17c61f.png">